src/test/components/MarkerChart.test.js

```diff
@@ -4,13 +4,16 @@
 
 // @flow
 import * as React from 'react';
-import { fireEvent } from '@testing-library/react';
```

Comment:
In some cases we were importing `fireEvent` from `'@testing-library/react'`, this wasn't a mistake but just less consistent. I moved it to the line with `firefox-profiler/test/fixtures/testing-library` where we reexport everything already (I added nothing there).